### PR TITLE
Make the return-type of `ClassReflection#getStartLine()` explicit

### PR DESCRIPTION
**Pull request by nicolas-grekas**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

This was removed in #154 but this phpdoc is useful to some static analyzers, so that they can know what the future return-type is going to be.

### CODE REVIEW

File: src/Reflection/ClassReflection.php [hunk 1]
```diff
@@ -44,7 +44,8 @@ public function getDocBlock()
     /**
      * {@inheritDoc}
      *
-     * @param bool $includeDocComment
+     * @param  bool $includeDocComment
+     * @return int|false
```
Comment (Ocramius) [inline src/Reflection/ClassReflection.php:48]: I would suggest trying `@inheritDoc` (no brackets) first.

Overall not opposed to this change, but to be attempted first.

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: src/Reflection/ClassReflection.php (hunk 1 above)
Comment (Ocramius) [reply src/Reflection/ClassReflection.php:48]: Wait... is this again because of Symfony's debug loader?

**Comment by IonBazan** [comment]
If we are aiming for `~8.1.0 || ~8.2.0` for v4.8.0, I think we don't really need to use `#[ReturnTypeWillChange]` and replace it with native union type there, right?

File: src/Reflection/ClassReflection.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Reflection/ClassReflection.php:48]: There's no `@inheritDoc` data to read on the parent (internal) class unfortunately so this doesn't provide the needed info.

**Comment by Ocramius** [comment]
@IonBazan changing a return type in a non-`final` class is still a BC break

File: src/Reflection/ClassReflection.php (hunk 1 above)
Comment (Ocramius) [reply src/Reflection/ClassReflection.php:48]: Fair enough: I still smell the stench of `symfony/debug`, but I'll let this slip :P

**Comment by IonBazan** [comment]
Noted, let's do that for v5, if you don't mind

**Comment by Ocramius** [comment]
@IonBazan I'd be really happy to make a lot of this stuff `final` for v5.

Once it's `final`, it is much much easier to maintain and change internals.

**Comment by nicolas-grekas** [comment]
Thanks for merging quickly!

**Comment by Ocramius** [comment]
No problem: just beware that I won't release until @IonBazan's #159 is also merged.

**Pull request merged**